src/scenes/ModulesRouter/ListView.tsx

```diff
@@ -13,6 +14,10 @@ import ListViewHeader from './ListViewHeader';
 export default function ListView(): JSX.Element {
   const theme = useTheme();
 
+  const { currentParticipant } = useParticipantData();
+
+  const isLoading = useMemo(() => !currentParticipant, [currentParticipant]);
```

Comment:
Is this needed? Can you just do `isLoading={!currentParticipant}` ?

Reply:
Fixed!